packages/ruleset/test/schema-requires-property.test.js

```diff
@@ -0,0 +1,171 @@
+const { schemaRequiresProperty } = require('../src/utils');
+
+describe('Utility function: schemaRequiresProperty()', () => {
+  it('should return `false` for `undefined`', async () => {
+    expect(schemaRequiresProperty(undefined, 'fungibility')).toBe(false);
+  });
+
+  it('should return `false` for `null`', async () => {
+    expect(schemaRequiresProperty(null, 'fungibility')).toBe(false);
+  });
+
+  it('should return `false` for empty schema', async () => {
+    expect(schemaRequiresProperty({}, 'fungibility')).toBe(false);
+  });
+
+  it('should return `true` for a compliant simple schema', async () => {
+    const compliantSimpleSchema = { required: ['fungibility'] };
```

Comment:
Yep, the property name is in the `required` list, but there's not actually a property named fungibility :)